CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
```

Comment:
In the commit title, the subject is not `main` but `edxapp`. It is only related to edxapp application.